staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -399,6 +440,19 @@ def onClick( self, control ):
             log("Button event: EXIT")
             self.exitXSqueeze()
 
+         #hdmi support
+          if (control == constants.BUTTONVOLUP) and constants.CECSUPPORT == 'true':
+            counter = 1
+            while counter <= int(constants.VOLFACTOR):
+             cec.volume_up()
+             counter = counter + 3
+
+          if (control == constants.BUTTONVOLDN)and constants.CECSUPPORT == 'true':
+            counter = 1
+            while counter <= int(constants.VOLFACTOR):
+             cec.volume_down()
+             counter = counter + 3
+
```

Comment:
Both of those should be for loops, not whiles - in pseudo

for (i=0; i < constants.VOLFVACTOR;  i++)